CHANGELOG.md

```diff
@@ -1,9 +1,37 @@
 # Changelog
 
 ## Unreleased
-* Fixed an issue where **lint** failed when using the `-cdam` flag with changed dependant api modules due to partial file duplications filtering.
+* Added a newline at the end of README files generated in **generate-docs**.
+* Added the value `3` (out of bounds) to the `onChangeRepAlg` and `reputationCalc` fields under the `IncidentType` and `GenericType` schemas. **validate** will allow using it now.
+* Added support for running **lint** in multiple native-docker images.
+* Fixed an issue where **doc-review** required dot suffixes in release notes describing new content.
 * Added the `--docker-image-target` flag to **lint** for testing native supported content with new images.
 
+
+## 1.10.4
+* Added support for running **lint** in multiple native-docker images.
+
+## 1.10.3
+* Fixed an issue where running **format** would fail after running npm install.
+* Improved the graph validations in the **validate** command:
+  - GR100 will now run on all content items of changed packs.
+  - GR101 and GR102 will now catch invalid fromversion/toversion of files **using** the changed items.
+  - GR103 errors will raise a warning when using the *-a* flag, but an error if using the *-i* or *g* flags.
+* Fixed an issue where test-playbooks timed out.
```

Comment:
why are all these part of your changes here?

Reply:
pre commit hook made merging difficult, I belive it is fixed now? where did you see this?

Reply:
looks fixed